### PR TITLE
[Fix] Add `rollout_logprobs` key to padding batches with dynamic micro batching 

### PR DESCRIPTION
**Pull request by SumanthRH**
^^^

### CODE REVIEW

File: skyrl/backends/skyrl_train/workers/worker_utils.py [hunk 1]
```diff
@@ -264,6 +264,9 @@ def _create_padding_microbatch(self) -> TrainingInputBatch:
                 "response_mask": torch.ones((batch_size, num_actions), dtype=int, device=device),
             }
         )
+        # Add optional fields such as `rollout_logprobs` to padding batch
+        if self.data.get("rollout_logprobs") is not None:
+            data["rollout_logprobs"] = torch.zeros((batch_size, num_actions), device=device)
```
[the remark this answers is not in the document]
Comment (SumanthRH) [reply skyrl/backends/skyrl_train/workers/worker_utils.py]: very very safe

[the remark this answers is not in the document]
Comment (SumanthRH) [reply skyrl/backends/skyrl_train/workers/worker_utils.py]: fixed now @gemini-code-assist
